### PR TITLE
Create example_openbsd.rst

### PR DESCRIPTION
**Pull request by gonzalo-**
Initial document for OpenBSD users, probably a lot of typos and missing parts, but it will be a great help so far.

### CODE REVIEW

**Comment by kesselb** [comment]
Thanks for your work 👍

Nextcloud does not support FreeBSD or OpenBSD. If we add a guide to the documentation people could assume that OpenBSD is supported.

**Comment by gonzalo-** [comment]
I am the port maintainer of Nextcloud on OpenBSD what kind of support are you talking about?

**Comment by kesselb** [comment]
Nextcloud is developed for Linux based operating systems. Most features should work on *BSD but it's not tested.

**Comment by gonzalo-** [comment]
Well that is a pity, anyway, if there is not other place on the official documents for people outside of Linux, you can close the ticket, thanks.

**Comment by kesselb** [comment]
I'm leaving it open. It's not my decision to merge or not. Probably good to merge with a disclaimer/warning.

**Comment by MorrisJobke** [comment]
> Probably good to merge with a disclaimer/warning.

I would be fine with a warning box at the top of the page. 👍 

@rullzer @karlitschek @jancborchardt @nickvergessen Opinions on this?

**Comment by karlitschek** [comment]
Yes. I think documentation about this is cool. The only thing we should avoid is to create the impression that it's working on BSD but in reality it's not. So if we are sure that is't working then let's add this documentation. If parts are not working we should add warnings. It is not working at all then no documentation please. I hope this makes sense :-)

**Comment by landryb** [comment]
There are lots of *BSD users, for which nextcloud *works*. Sure there are some sorry linuxisms here and there, but *BSD users generally dont expect 'support' from upstream, rather from their packager. Giving them the middle finger in the documentation is not a nice solution either, so maybe some middle ground ? ;)

(yeah, using NC 19 on OpenBSD 6.7, it generally works.)

**Comment by gonzalo-** [comment]
> The only thing we should avoid is to create the impression that it's working on BSD but in reality it's not.

It works without any big issue, maybe you want to say "We have no official *BSD support*?

Sounds more precise to me.

**Comment by MorrisJobke** [comment]
> It works without any big issue, maybe you want to say "We have no official _BSD support_?

Yep - that sounds good IMO 👍

**Comment by gonzalo-** [comment]
So should I do something or you will commit this with the warning we talked about?

**Comment by kesselb** [comment]
>  missing parts

What parts are missing?

File: admin_manual/installation/example_openbsd.rst [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+		  pass
+	  }
+
+	  location "/nextcloud/core/*" {
+		  root "/nextcloud"
+		  request strip 1
+		  pass
+	  }
+
+	  location "/nextcloud/settings/*" {
```
Comment (kesselb) [inline admin_manual/installation/example_openbsd.rst:70]: This section is not longer necessary. Settings is nowadays an app in /nextcloud/apps.

File: admin_manual/installation/example_openbsd.rst [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+    ),
+    ...
+::
+
+Cron job
+--------
+
+We need to add the Nextcloud cron job to get some tasks done by adding this entry on your cronjob::
+
+  */15 * * * * /usr/bin/ftp -Vo - https://domain.tld/cron.php >/dev/null
```
Comment (kesselb) [inline admin_manual/installation/example_openbsd.rst:199]:
```suggestion
  */5 * * * * /usr/bin/ftp -Vo - https://domain.tld/cron.php >/dev/null
```

We also suggest to run this from CLI.

File: admin_manual/installation/example_openbsd.rst (hunk 1 above)
Comment (gonzalo-) [reply admin_manual/installation/example_openbsd.rst:70]: Cool, good to know.

File: admin_manual/installation/example_openbsd.rst (hunk 2 above)
Comment (gonzalo-) [reply admin_manual/installation/example_openbsd.rst:199]: Yes, but usually httpd(8) runs under a chroot() so for some users is more useful to just put a cronjob.

**Comment by gonzalo-** [comment]
> What parts are missing?

Nothing really important, just cosmetic things, but you can have NC working on OpenBSD with this tutorial plus our own README file.

**Comment by gonzalo-** [comment]
@kesselb @MorrisJobke  Can we have this in?

**Comment by kesselb** [comment]
Can you fix the linter warnings?

**Comment by gonzalo-** [comment]
@kesselb do you mean the cron and the "location "/nextcloud/settings/*" {" things? I changed it already, maybe I did it wrong :/

**Comment by kesselb** [comment]
image
image
image
image
image

**Comment by gonzalo-** [comment]
@kesselb better now? :D

**Comment by kesselb** [comment]
@gonzalo- thanks :+1: 

image

To fix the above warning we need to reference the guide: https://github.com/nextcloud/documentation/blob/master/admin_manual/installation/index.rst.

image

I don't know how to fix this warning but it's not a blocker.

image

Please sign the commits: https://github.com/nextcloud/documentation/pull/5039/checks?check_run_id=1925653519

**Comment by gonzalo-** [comment]
@kesselb cool! 

thanks for the hint! Already open another PR with that, and fixed the other one (I think).

**Pull request closed without merging**